### ISSUE TITLE
Stories

### ISSUE THREAD

**Issue by startupgurukul** [adityadroid/Messio#44]
Hey Aditya congrats on almost reaching the end, do you plan to implement stories as well?

**Comment by adityadroid** [comment]
> Hey Aditya congrats on almost reaching the end of you 60 day challenge, do you plan to implement stories as well?

No. I might do it in future. But as of now I haven't thought of doing another phase for the app.
